lib/Mojo/Server/Daemon.pm

```diff
@@ -194,7 +194,8 @@ sub _listen {
       my ($loop, $stream, $id) = @_;
 
       $self->{connections}{$id} = {tls => $tls};
-      warn "-- Accept $id (@{[$stream->handle->peerhost]})\n" if DEBUG;
+      my $peer = $stream->handle->isa('IO::Socket::UNIX') ? 'peerpath' : 'peeraddr' if DEBUG;
+      warn "-- Accept $id (@{[$stream->handle->$peer]})\n" if DEBUG;
```

Comment:
This code is rather ugly. Duplicated calls of`$stream->handle` and uses of the `DEBUG` constant.

Reply:
Force pushed an update to address your comments.

Reply:
@kraih After sleeping on it, a function is much nicer.  I have force-pushed a better solution to this PR.